### PR TITLE
Updating Copyright Cypress test to be include BBC Copyright

### PR DESCRIPTION
**Pull request by jamesbrumpton**
**Overall change:** _Currently, our Copyright tests only test for visible image copyright. However, BBC copyright images have no visible copyright. Whilst there's no visible copyright we can still check to make sure that when the copyright isn't visible that it is returning BBC copyright._

**Code changes:**

- _Changing the command `copyrightDataWindow`._
- _if/else block used to see whether a `<p>` is present within the `figure`._
- _If `<p>` is present `copyrightHolder` data is compared, as per the previous test._
- _If `<p>` isn't present, we assert that the data within `copyrightHolder` is `'BBC'`._
---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: cypress/integration/pages/article/canonical.js [hunk 1]
```diff
@@ -321,8 +321,9 @@ Object.keys(config)
 
           it('should have an image copyright label with styling', () => {
             cy.copyrightDataWindow();
-          });
+          })
         }
+
```
Comment (12) [inline cypress/integration/pages/article/canonical.js]:
```suggestion
```

File: cypress/integration/pages/article/canonical.js [hunk 2]
```diff
@@ -321,8 +321,9 @@ Object.keys(config)
 
           it('should have an image copyright label with styling', () => {
             cy.copyrightDataWindow();
-          });
+          })
```
Comment (12) [inline cypress/integration/pages/article/canonical.js]:
```suggestion
          });
```

**Review by 12 — CHANGES_REQUESTED** [review]

**Review by 12 — APPROVED** [review]
The comment makes things very clear, thanks 👍

**Review by RayNjeri — APPROVED** [review]
LGTM

**Review by j-pendlebury — APPROVED** [review]

**Pull request merged**